mobile/evidence/ios/Leaf/Sources/Leaf/Theme/LeafEnvironment.swift

```diff
@@ -51,6 +57,12 @@ public let darkLeafTheme = LeafTheme(
             secondary: .gray,
             tertiary: .gray,
             quaternary: .green
+        ),
+        auxiliar: LeafTheme.AuxiliarColor(
+            auxiliar: Color(red:74/255, green: 179/255, blue: 255/255),
+            success:  Color(red:68/255, green: 177/255, blue: 16/255),
+            error: Color(red:249/255, green: 100/255, blue: 90/255),
+            disabled: Color(red:162/255, green: 168/255, blue: 159/255)
```

Comment:
I think `accepted`, `rejected`, `open` are important concepts in the specific domain of our application. Maybe we should have color tokens for those concepts?

Reply:
So,  those colors are exactly for those concepts.  Auxiliary: open, success: accepted, error: rejected, and disabled: closed. I will change the names. I created like this bc I thoughts reuse like error, success, etc.

Reply:
```swift

``` tag: LeafTheme.TagColor(
            open: Color(red:3/255, green: 129/255, blue: 220/255),
            accepted: .green,
            rejected: Color(red:249/255, green: 100/255, blue: 90/255),
            closed: Color(red:121/255, green: 128/255, blue: 117/255)
        )

Reply:
Eu fiz essas alterações :D